doc/INSTALL.md

```diff
@@ -276,6 +277,37 @@ need to include `testnet=1`
     ./lightningd/lightningd &
     ./cli/lightning-cli help
 
+To Build on Arch Linux
+---------------------
+
+Install dependencies:
+
+```
+pacman --sync autoconf automake gcc git make python-pip
+pip install --user poetry
+```
+
+Clone Core Lightning:
+
+```
+$ git clone https://github.com/ElementsProject/lightning.git
+$ cd lightning
+```
+
+Build Core Lightning:
+
+```
+~/.local/bin/poetry install
+./configure
+~/.local/bin/poetry run make
+```
+
+Launch Core Lightning:
+
+```
+./lightningd/lightningd
+```
+
```

Comment:
```suggestion
Launch Core Lightning:

python -m poetry shell
sudo make install


```

Maybe poetry shell can be avoided? here not sure, but you can try.

The motivation of this is that the other section use `make install`, so we need to be consistent with our docs

N.B the code syntax need to stay, github has some bugs with the suggestion feature, sorry :)